srcpkgs/cpufetch/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'cpufetch'
+pkgname=cpufetch
+version=1.05
+revision=1
+archs="i686* x86_64*"
```

Comment:
https://github.com/Dr-Noob/cpufetch#1-support

Reply:
See comment:
> In theory it should work on `ppc*` and `arm*`, but it won't build successfully for me due to the `-mavx*` flag.

For example, `$ ./xbps-src -a armv7l pkg cpufetch`
```
armv7l-linux-gnueabihf-gcc -fstack-clash-protection -D_FORTIFY_SOURCE=2 -O2 -pipe -march=armv7-a -mfpu=vfpv3 -mfloat-abi=hard   -I/usr/armv7l-linux-gnueabihf/usr/include -ffile-prefix-map=/builddir/cpufetch-1.05=. -Wall -Wextra -pedantic -pthread -DARCH_X86 -std=c99 -fstack-protector-all -O2 -Wfloat-equal -Wshadow -Wpointer-arith -Wstrict-prototypes -c -pthread src/x86/freq/freq_nov.c -o freq_nov.o
armv7l-linux-gnueabihf-gcc -fstack-clash-protection -D_FORTIFY_SOURCE=2 -O2 -pipe -march=armv7-a -mfpu=vfpv3 -mfloat-abi=hard   -I/usr/armv7l-linux-gnueabihf/usr/include -ffile-prefix-map=/builddir/cpufetch-1.05=. -Wall -Wextra -pedantic -pthread -DARCH_X86 -std=c99 -fstack-protector-all -O2 -Wfloat-equal -Wshadow -Wpointer-arith -Wstrict-prototypes -c -mavx -pthread src/x86/freq/freq_avx.c -o freq_avx.o
armv7l-linux-gnueabihf-gcc -fstack-clash-protection -D_FORTIFY_SOURCE=2 -O2 -pipe -march=armv7-a -mfpu=vfpv3 -mfloat-abi=hard   -I/usr/armv7l-linux-gnueabihf/usr/include -ffile-prefix-map=/builddir/cpufetch-1.05=. -Wall -Wextra -pedantic -pthread -DARCH_X86 -std=c99 -fstack-protector-all -O2 -Wfloat-equal -Wshadow -Wpointer-arith -Wstrict-prototypes -c -mavx512f -pthread src/x86/freq/freq_avx512.c -o freq_avx512.o
armv7l-linux-gnueabihf-gcc: error: unrecognized command-line option '-mavx'
make: *** [Makefile:84: freq_avx.o] Error 1
make: *** Waiting for unfinished jobs....
armv7l-linux-gnueabihf-gcc: error: unrecognized command-line option '-mavx512f'
make: *** [Makefile:87: freq_avx512.o] Error 1
src/x86/freq/freq_nov.c:2:10: fatal error: immintrin.h: No such file or directory
    2 | #include <immintrin.h>
      |          ^~~~~~~~~~~~~
compilation terminated.
```

Reply:
try adding `arch=${XBPS_TARGET_MACHINE%-musl}` to `make_build_args`

Reply:
That's fixed it, thanks! The only things it now won't build on are `armv5*` and `mips*` which are unsupported anyways. I'll push the change.